be/src/vec/columns/column_nullable.cpp

```diff
@@ -570,6 +570,16 @@ ColumnPtr remove_nullable(const ColumnPtr& column) {
     if (is_column_nullable(*column)) {
         return reinterpret_cast<const ColumnNullable*>(column.get())->get_nested_column_ptr();
     }
+
+    if (is_column_const(*column)) {
```

Comment:
Why not put this in a new method in column_const.h?

Reply:
here two method `add_remove` nullable bosh in `nullable.cpp`